tools/backups2datalad.py

```diff
@@ -611,79 +432,237 @@ def describe_dandiset(self, dandiset_id):
+        git(
+            "tag",
+            "-m",
+            f"Version {dandiset.version_id} of Dandiset {dandiset.identifier}",
+            dandiset.version_id,
+            commitish,
         )
+        # if branching:
+        #     git("push", "-u", "github", "release-{dandiset.version_id}")
+        # git("push", "--follow-tags", "github")
+        ds.push(to="github", jobs=self.jobs)
+        if branching:
+            git("checkout", "master")
+
+    @cached_property
+    def s3client(self) -> S3Client:
+        return boto3.client("s3", config=Config(signature_version=UNSIGNED))
+
+    @cached_property
+    def gh(self) -> Github:
+        token = readcmd("git", "config", "hub.oauthtoken")
+        return Github(token)
+
+
+@click.group()
+@click.option(
+    "--asset-filter",
+    help="Only consider assets matching the given regex",
+    metavar="REGEX",
+)
+@click.option(
+    "-A",
+    "--assetstore",
+    type=click.Path(exists=True, file_okay=False, path_type=Path),
+    required=True,
+)
+@click.option(
+    "-i",
+    "--dandi-instance",
+    type=click.Choice(sorted(known_instances)),
+    default="dandi",
+    help="DANDI instance to use",
+    show_default=True,
+)
+@click.option(
+    "-J",
+    "--jobs",
+    type=int,
+    default=10,
+    help="How many parallel jobs to use when pushing",
+    show_default=True,
+)
+@click.option("-f", "--force", type=click.Choice(["check"]))
+@click.option("--ignore-errors", is_flag=True)
+@click.option(
+    "-l",
+    "--log-level",
+    type=LogLevel(),
+    default="INFO",
+    help="Set logging level  [default: INFO]",
+)
+@click.option("--pdb", is_flag=True, help="Drop into debugger if an error occurs")
+@click.option(
+    "-T", "--target", type=click.Path(file_okay=False, path_type=Path), required=True
+)
+@click.pass_context
+def main(
+    ctx: click.Context,
+    asset_filter: str,
+    assetstore: Path,
+    dandi_instance: str,
+    force: Optional[str],
+    ignore_errors: bool,
+    jobs: int,
+    log_level: int,
+    pdb: bool,
+    target: Path,
+) -> None:
+    ctx.obj = DandiDatasetter(
+        dandi_client=ctx.with_resource(
+            DandiAPIClient.for_dandi_instance(dandi_instance)
+        ),
+        assetstore_path=assetstore,
+        target_path=target,
+        ignore_errors=ignore_errors,
+        asset_filter=re.compile(asset_filter) if asset_filter is not None else None,
+        jobs=jobs,
+        force=force,
+    )
+    if pdb:
+        sys.excepthook = pdb_excepthook
+    logging.basicConfig(
+        format="%(asctime)s [%(levelname)-8s] %(name)s %(message)s",
+        datefmt="%Y-%m-%dT%H:%M:%S%z",
+        level=log_level,
+        force=True,  # Override dandi's settings
+    )
+
+
+@main.command()
+@click.option("--backup-remote", help="Name of the rclone remote to push to")
+@click.option(
+    "-e", "--exclude", help="Skip dandisets matching the given regex", metavar="REGEX"
+)
+@click.option(
+    "--gh-org",
```

Comment:
is there an easy way to make this optional and if not specified, do not bother with any "github" related operations (such as push?). Asking because 
- if to quickly test the script on staging, I would probably prefer not bother pushing any results anywhere.  If I really decide to test, I would need to point to some other organization than dandisets I guess. (actually -- we might better get `dandisets-staging` organization so we could indeed test a full cycle etc on staging
- whenever tests are to be added, I guess they would also need to avoid interactions with github

Reply:
Made --gh-org optional for update-from-backup (Naturally, it's required for update-github-metadata).